src/applications/mhv-secure-messaging/tests/e2e/sort-filter-test/secure-messaging-add-filter-custom-date-custom-folder.cypress.spec.js

```diff
@@ -25,17 +25,17 @@ describe('SM CUSTOM FOLDER ADD FILTER CUSTOM DATE RANGE', () => {
   });
 
```

Comment:
mhv changes are due to the date-field being updated to remove blank/null options